src/engine/controls/cuecontrol.cpp

```diff
@@ -276,15 +280,40 @@ void CueControl::createControls() {
 
         m_hotcueControls.append(pControl);
     }
+
+    for (int i = 0; i < m_iNumSavedLoops; ++i) {
+        SavedLoopControl* pControl = new SavedLoopControl(getGroup(), i);
+
+        connect(pControl, &SavedLoopControl::savedLoopPositionChanged,
+                this, &CueControl::savedLoopPositionChanged,
+                Qt::DirectConnection);
+        connect(pControl, &SavedLoopControl::savedLoopLengthChanged,
+                this, &CueControl::savedLoopLengthChanged,
+                Qt::DirectConnection);
+        connect(pControl, &SavedLoopControl::savedLoopSet,
+                this, &CueControl::savedLoopSet,
+                Qt::DirectConnection);
+        connect(pControl, &SavedLoopControl::savedLoopActivate,
+                this, &CueControl::savedLoopActivate,
+                Qt::DirectConnection);
+        connect(pControl, &SavedLoopControl::savedLoopReloop,
+                this, &CueControl::savedLoopReloop,
+                Qt::DirectConnection);
+        connect(pControl, &SavedLoopControl::savedLoopClear,
+                this, &CueControl::savedLoopClear,
+                Qt::DirectConnection);
+
+        m_savedLoopControls.append(pControl);
```

Comment:
You need to delete the controls in the destructor. You may use qDeleteAll (old style), or use a std::vector and uniqe_ptr()
https://github.com/mixxxdj/mixxx/blob/9598072229678dea8ffefec3826490f5806bfaea/src/effects/builtin/parametriceqeffect.h#L31